composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": ">=5.4.0",
         "lib-pcre": ">=7.0",
-        "hamcrest/hamcrest-php": "~1.1"
+        "hamcrest/hamcrest-php": "~2.0-dev"
```

Comment:
The constraint should say `^2.0@dev` instead. This way:

* you will be checking for a regular version
* for Mockery contributors the dev version of Hamcrest will be used
* for everybody else (who include Mockery as a dependency) stable 2.x version of Hamcrest will be used